docs/en/alab/project_dashboard.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: docs
+comment: no
+header: true
+seotitle: Annotation Lab | John Snow Labs
+title: Dashboard 
+permalink: /docs/en/alab/project_dashboard
+key: docs-training
+modify_date: "2022-10-19"
+use_language_switcher: "Python-Scala"
+show_nav: true
+sidebar:
+    nav: annotation-lab
+---
+
+When you login to the Annotation Lab, you will be presented with the the project dashboard. For each project details like description, task counts, groups, team members, etc. are available on the main dashboard so users can quickly identify the projects they need to work on, without navigating to the Project Details page. You will be presented with the list of available projects you have created **My Projects** or that have been shared with you **Shared With Me**. **All Projects** combines the list of the projects created by you and the the projects shared to the you.
+
+<img class="image image--xl" src="/assets/images/annotation_lab/4.1.0/dashboardShared.png" style="width:100%; align:center; box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);"/>
+
+
+The list of projects can be sorted according to the name of the project. Also, projects can be sorted in ascending or descending order according to the creation date. name. 
+
+<img class="image image--xl" src="/assets/images/annotation_lab/4.1.0/dashboardSort.png" style="width:100%; align:center; box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);"/>
```

Comment:
You can remove the `align:center;` from the style attribute as it is incorrect property field. And instead of `box-shadow: ....` you can add the class `image__shadow` to the class attribute.